src/simplewallet/simplewallet.cpp

```diff
@@ -2205,6 +2206,24 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
 
   std::vector<std::string> local_args = args_;
 
+  int priority = 0;
+  if(local_args.size() > 0) {
+    bool priority_found = false;
+    for(auto& priority_string : allowed_priority_strings) {
+      if(local_args[0] == priority_string) {
+        priority_found = true;
+        break;
+      } else {
+        priority++;
+      }
+    }
+    if(priority_found) {
+      local_args.erase(local_args.begin());
+    } else {
+      priority = 0;
+    }
+  }
+
```

Comment:
This whole thing can be replaced with calls to std::find and std::distance I think.